lib/upnp/igd_service.go

```diff
@@ -120,8 +226,13 @@ func (s *IGDService) GetExternalIPAddress(ctx context.Context) (net.IP, error) {
 }
 
 // GetLocalIPAddress returns local IP address used to contact this service
```

Comment:
Docs should match function name.
```suggestion
// GetLocalIPv4Address returns local IP address used to contact this service
```